File: YEAR2/SEM2/Medii de proiectare si programare/ConsoleApp1/repository0/IOficiuPersoana.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ConsoleApp1.domain;
using ConsoleApp1.repository;

namespace ConsoleApp1.repository0
{
    public interface IOficiuPersoana : IRepository<long,OficiuPersoana>
    {
        public OficiuPersoana LogIn(String user, String password);
    }
}

File: YEAR2/SEM2/Medii de proiectare si programare/Tema1/src/main/java/org/example/repository/database/IOficiuPersoana.java
package org.example.repository.database;

import org.example.domain.OficiuPersoana;
import org.example.repository.Repository0;

public interface IOficiuPersoana extends Repository0<Long, OficiuPersoana> {

    OficiuPersoana Login(String user,String password);

}
